src/radchallenge.c

```diff
@@ -0,0 +1,210 @@
+/*
+ * radchallenge.c - a sample c program showing how to embed the configuration of a radius and to challenge
+ * client, using the FreeRADIUS Client Library without an external configuration and dictionary file .
```

Comment:
Here it mentions the freeradius client library, I think it should mention radcli instead